dashboard/app/views/levels/edit.html.haml

```diff
@@ -25,6 +25,7 @@
         %p the level cannot be renamed because it is a standalone project level.
   .field
     = render partial: 'levels/editors/fields/checkboxes', locals: {f: f, field_name: :encrypted, description: "Encrypt this level's properties"}
+    = render partial: 'levels/editors/fields/checkboxes', locals: {f: f, field_name: :neighborhood_enabled, description: "Display Neighborhood visualization"}
```

Comment:
Does this add the setting to all levels in levelbuilder, not just javalab ones? I've been messing around in [this file](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/views/levels/editors/_javalab.html.haml) for adding settings just to the javalab levels and [this folder](https://github.com/code-dot-org/code-dot-org/tree/staging/dashboard/app/views/levels/editors/fields) for adding a new field

Reply:
Oh nice thanks for this! Taking a look now.